tests/unit/test_eel.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+import eel
+import pytest
+from tests.utils import TEST_DATA_DIR
+
+INIT_DIR = TEST_DATA_DIR / 'init_test'
+"""Directory for testing eel.__init__."""
```

Comment:
```suggestion
# Directory for testing eel.__init__.
INIT_DIR = TEST_DATA_DIR / 'init_test'
```

Just a minor nit to have the comment before the related code and use `#` for non-docstring comments.